Menta themes: set color to the notebook header border

sc0w opened the pull request:
after the commit https://github.com/mate-desktop/mate-themes/commit/15152c6c5716464fba57e44c324a168e9a5a3101 we have this issue:

![image4493](https://cloud.githubusercontent.com/assets/7734191/24481575/746bde6c-14eb-11e7-93cd-dd0ce7319e5b.png)

If we move the left tab to the right

This is the fix for menta themes, using the exactly color needed, I don't know if this color is defined with some name

I don't know how to do for traditional themes, I need to use a gradient color for the border

Probably more themes are affected, I didn't test all

raveit65 commented:
damned, why it is allowed to move the tab to right in result we see the header there......

raveit65 commented:
Hmm, how did you did that?
Here the active or inactive tab are always at the left edge/right of the header after dragging. 
They use always the full given space....here.

sc0w commented:
I just move the first tab, easy to reproduce, see the video:

[out-24.ogv.zip](https://github.com/mate-desktop/mate-themes/files/881524/out-24.ogv.zip)

raveit65 commented:
yes, but the tap snap back to normal position after dropping with the mouse.
Sounds like a very theoretical issue :-)
I will merge it like this
```
window.background.mate-terminal > box.vertical > notebook > header.top {
    border-width: 0px 0px 0px 1px;
    border-color: @theme_bg_color;
}
```
I prefer to use color-definitions, and i will see what i can do for Trad themes.

sc0w commented:
> yes, but the tap snap back to normal position after dropping with the mouse.

yes

> Sounds like a very theoretical issue :-)

We must to obtain the perfection :)

> I prefer to use color-definitions

me too, I didn't know the color-definition for this

commit rebased

> i will see what i can do for Trad themes.

thanks

raveit65 commented:
merged 2927783fb715251b27e41d457c39b9896ab1d130
and same for Traditional themes
https://github.com/mate-desktop/mate-themes/commit/5259a5d36fc063382141eafe9da5ef315cd64e98
Thank you

sc0w commented:
the commit https://github.com/mate-desktop/mate-themes/commit/5259a5d36fc063382141eafe9da5ef315cd64e98 is wrong

you added **@theme_bg_color** to **header.top** in TG, but to **header.top tab** in TOK

I fixed and added a gradient color to the border instead: https://github.com/mate-desktop/mate-themes/commit/44237d051e7a0dacd16c81bef28e45367064c76d

The same gradient than **notebook header.top** in **gtk-widgets.css**

https://github.com/mate-desktop/mate-themes/blob/master/desktop-themes/TraditionalOk/gtk-3.0/gtk-widgets.css#L1761
